src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+public class Task {
+    protected String description;
```

Comment:
In other classes that inherit `Task` class, you leave a line before declaring the attributes. Here there isn't a separate blank line before `description'. you could ensure that the spacing is the same across your code.